src/js/feedback/containers/Main.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { sendFeedback, clearError } from '../actions';
+import DefaultView from '../components/DefaultView';
+import FeedbackForm from '../components/FeedbackForm';
+import FeedbackSubmitted from '../components/FeedbackSubmitted';
+
+class Main extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { formIsVisible: false, requestPending: false };
+  }
+
+  revealForm = () => {
+    this.setState({ formIsVisible: true });
+  }
+
+  render() {
+    let content = null;
+
+    if (this.props.feedbackReceived) {
+      content = (
+        <FeedbackSubmitted
+          shouldSendResponse={this.props.shouldSendResponse}/>
+      );
+    } else if (!this.state.formIsVisible) {
```

Comment:
I'm curious how a screen reader would deal with the trigger for showing the form being completely replaced. In other places where we have collapsing/expanding content we use aria-expanded and aria-controls to convey some of that.